Add blog post with info about DMCA (#330)
TeamNewPipe/website: _posts/2023-07-07-newpipe-net-dmca-google-search.md

---
layout: post
title: "newpipe.net removed from Google search results due to DMCA take down request"
short: "newpipe.net removed from Google search results"
date: 2023-07-07 15:00:00 + 02:00
categories: [pinned, announcement]
author: <a href="https://github.com/theScrabi">@theScrabi</a>
image: newpipe
excerpt_separator: <!-- more -->
---
Dear NewPipe community and other beloved creatures,

> Maybe you have already noticed this: if you search for "newpipe" via Google, you will find everything about us, except the official website. This is because Google accepted a DMCA request from the French company "Because Music" against our homepage, which means that from now on the landing page at newpipe.net will not show up in Google search results. You can find the full DMCA [here in the Lumen database](https://lumendatabase.org/notices/34149383).  

Reviewer: ```suggestion
Maybe you have already noticed this: if you search for "NewPipe" via Google, you will find plenty information about the project, but a link to the official website is missing. This is because Google submitted to a DMCA takedown notice from a French record label, "Because Music". The notice demands removing our homepage [newpipe.net](https://newpipe.net) from their search listings along with a set of other domains completely unrelated to our project. The DMCA takedown notice was published [in the Lumen database](https://lumendatabase.org/notices/34149383) by Google themselves.
```

Branding is important, so please use the right spelling.

> While this is not the worst thing that could have happened, it is still rather nasty. Because the official website is hidden in the search results, fake clones of the site will tend to rank higher in the results, and our site may lose relevance. We are currently discussing whether to take legal action against this DMCA.

Reviewer: ```suggestion
While worse things could have happened as the result of such a takedown request, Google's unlisting of our homepage creates a variety of problems. Given the fact that Google no longer counts visits to our site, it lose relevance. As a result, websites for fake clones of NewPipe that mimic our homepage will tend to rank higher in the results. This could ultimately cause users to fall for such scams.
```

> If you are a lawyer and have experience with this and would like to help us, please contact us via email. (team(at)newpipe.net)

Reviewer: ```suggestion
The team is currently discussing what to do about the situation. We consider taking legal action. **If you are a lawyer and have experience with this and would like to help us, please contact us via email (team(at)newpipe.net).**

We are going to keep you updated on the issue in future blog posts.
```

---
layout: post
title: "newpipe.net removed from Google search results due to DMCA take down request"
short: "newpipe.net removed from Google search results"
date: 2023-07-07 15:00:00 + 02:00
categories: [pinned, announcement]
author: <a href="https://github.com/theScrabi">@theScrabi</a>
image: newpipe
excerpt_separator: <!-- more -->
---
Dear NewPipe community and other beloved creatures,

> Maybe you have already noticed this: if you search for "NewPipe" via Google, you will find plenty information about the project, but a link to the official website is missing. This is because Google submitted to a DMCA takedown notice from a French record label, "Because Music". The notice demands removing our homepage [newpipe.net](https://newpipe.net) from their search listings along with a set of other domains completely unrelated to our project. The DMCA takedown notice was published [in the Lumen database](https://lumendatabase.org/notices/34149383) by Google themselves.

Reviewer 2: ```suggestion
You might have already noticed this: Searching "NewPipe" via Google will yield plenty information about the project, but a link to the official website is missing. This is because Google submitted to a DMCA takedown notice from a French record label, "Because Music". The notice demands removing our homepage [newpipe.net](https://newpipe.net) from their search listings along with a set of other domains completely unrelated to our project. The DMCA takedown notice was published [in the Lumen database](https://lumendatabase.org/notices/34149383) by Google themselves.
```
